src/dotnet/APIView/APIViewWeb/APIViewWeb.csproj

```diff
@@ -25,8 +25,12 @@
     <PackageReference Include="AutoMapper.Extensions.Microsoft.DependencyInjection" Version="11.0.0" />
     <PackageReference Include="Azure.Storage.Blobs" Version="12.13.0" />
     <PackageReference Include="CsvHelper" Version="27.2.1" />
-    <PackageReference Include="Markdig" Version="0.30.3" />
-    <PackageReference Include="Markdig.SyntaxHighlighting" Version="1.1.7" />
+    <PackageReference Include="Markdig" Version="0.30.3">
+      <NoWarn>NU1701</NoWarn>
+    </PackageReference>
+    <PackageReference Include="Markdig.SyntaxHighlighting" Version="1.1.7">
+      <NoWarn>NU1701</NoWarn>
+    </PackageReference>
```

Comment:
Just leave a comment that explains why this suppression is required, e.g. "Adding NU1701 suppression as these two libraries have not updated to more modern versions of .net, although in our testing they work as required"